app/components/home-page.js

```diff
@@ -44,6 +62,30 @@ export default Ember.Component.extend({
     this.securityEnabled();
   },
 
+  /* Update org name */
+  updateOrgName: task(function * () {
+    const org = this.get('organization').selected;
+    org.set('name', this.get("organizationName"));
+    yield org.save();
+  }).evented(),
+
+  updateOrgNameSucceeded: on('updateOrgName:succeeded', function() {
+    this.get('notify').success(this.get('tOrganizationOrgNameUpdated'));
```

Comment:
Typo here ^ it is `tOrganizationNameUpdated`

Reply:
shit 🙈